test/jasmine/tests/hover_label_test.js

```diff
@@ -499,6 +499,43 @@ describe('hover info', function() {
             .catch(fail)
             .then(done);
         });
+
+        it('should display correct label content with specified format', function(done) {
```

Comment:
Looks like the two test cases you added are pretty much equivalent. Pick your favorite and :hocho: the other.